lib/screens/components/warnings_list.dart

```diff
@@ -0,0 +1,37 @@
+import 'package:flutter/material.dart';
+
+class WarningsList extends StatelessWidget {
+  final List warnings;
+
+  WarningsList({this.warnings});
+
+  @override
+  Widget build(BuildContext context) {
+    return Column(
+      children: <Widget>[
+        new Padding(
+          padding: new EdgeInsets.only(top: 20.0),
+        ),
+        new Expanded(
+          child: new ListView.builder(
+            itemCount: warnings.length,
+            itemBuilder: (BuildContext context, int index) {
+              var warning = warnings[index];
+              return Padding(
+                padding: const EdgeInsets.only(bottom: 10.0),
+                child: ListTile(
```

Comment:
i know the structure for Madeira warnings is a bit different, but would be great if the timestamp was shown on Madeira Warnings list (for the other warnings, the timestamp is the Warning title)